src/include/compat.h

```diff
@@ -280,13 +280,19 @@ int lchown(const char *path, uid_t owner, gid_t group);
 
 #ifdef __cplusplus
 }
+
+// Windows' mkdir doesn't accept a mode argument.
+#define compat_mkdir(pathname, mode) mkdir(pathname)
```

Comment:
could you instead define a `mkdir()` for Windows instead of having a wrapper for `mkdir()`.

Reply:
``mkdir`` is already defined on Windows but has a different signature.

Reply:
i see. probably we need to use `std::filesystem::create_directory()` in future.

Reply:
#36053